mmv1/products/apigee/api.yaml

```diff
@@ -206,4 +206,60 @@ objects:
       guides:
         'Creating a runtime instance':
           'https://cloud.google.com/apigee/docs/api-platform/get-started/create-instance'
-      api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances/create'
\ No newline at end of file
+      api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances/create'
+  - !ruby/object:Api::Resource
+    name: 'Environment'
+    base_url: 'environments'
+    create_url: '{{org_id}}/environments'
+    self_link: '{{org_id}}/environments/{{name}}'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: True
+        allowed:
+          - True
+          - False
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    description: |
+      An `Environment` in Apigee.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'orgId'
```

Comment:
It looks like the API docs specify this as `parent` - why does `orgId` work? https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments/create

Reply:
parent is just a URL parameter, so we can call it however we want. We actually have the two models in other resources. For example, the older [`google_access_context_manager_access_level`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/access_context_manager_access_level) used parent as is, while the newer [`google_kms_crypto_key`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/kms_crypto_key) uses the parent as `key_ring`. The second (with an explicit field name that matches what is expected) seems more readable.